### ISSUE TITLE
Not created milter socket file.

### ISSUE THREAD

**Issue by mzch** [roehling/postsrsd#136]
I built postsrsd with milter support, but postsrsd doesn't create a milter socket file.

- cmake options:
```
cmake .. -DMAKE_BUILD_TYPE=Release -DCMAKE_INSALL_PREFIX=/usr/local -DCMAKE_INSTALL_SYSCONFDIR=/etc/postsrsd -DPOSTSRSD_USER=postsrsd -DPOSTSRSD_DATADIR=/var/lib/postsrsd -DWITH_SQLITE=ON -DWITH_REDIS=ON -DWITH_MILTER=ON -DBUILD_TESTING=OFF
```

- postsrsd.conf
```
socketmap = unix:/var/spool/postfix/run/srs
…
milter = unix:/var/spool/postfix/run/srs_milter
```

**Comment by roehling** [comment]
I think this may be caused by the way I interact with LibMilter, because the socket might be created only *after* I dropped root privileges, which makes it likely impossible to write to `/var/spool/postfix`. Can you try and create a `postsrsd` subdir there with the correct permissions for the unprivileged `postsrsrd` user to write, and create the milter socket there?

**Comment by roehling** [comment]
This should be fixed by 1bf5c369a791f6bdfd28944597f9b7e870954133
